src/persistQueryClient/persist.ts

```diff
@@ -156,17 +152,14 @@ export function persistQueryClient(
     persistQueryClientUnsubscribe?.()
   }
 
+  // Attempt restore
   let restorePromise = Promise.resolve()
```

Comment:
we can now get rid of the let, as it was only necessary so that we have a return value if there is no window.